php/src/Docker/DockerActionManager.php

```diff
@@ -425,10 +409,18 @@ public function CreateContainer(Container $container) : void {
             $requestBody['HostConfig']['SecurityOpt'] = ["apparmor:unconfined"];
         }
 
+        $mounts = [];
+        foreach($container->GetVolumes()->GetVolumes() as $volume) {
+            if (!str_starts_with($volume->name, '/')) {
+                $mounts[] = ["Type" => "volume", "Source" => $volume->name, "Target" => $volume->mountPoint, "ReadOnly" => !$volume->isWritable];
+            } else {
+                $mounts[] = ["Type" => "bind", "Source" => $volume->name, "Target" => $volume->mountPoint, "ReadOnly" => !$volume->isWritable, "Propagation" => "rshared"];
```

Comment:
Should try if that works and creates not existing dirs as well
```suggestion
                $mounts[] = ["Type" => "volume", "Source" => $volume->name, "Target" => $volume->mountPoint, "ReadOnly" => !$volume->isWritable, "Propagation" => "rshared"];
```

Reply:
Or maybe just check first if it creates nor existing dirs first